tensorflow/lite/examples/minimal/minimal.cc

```diff
@@ -53,8 +53,8 @@ int main(int argc, char* argv[]) {
   // which allocates memory for the Intrepter and does various set up
   // tasks so that the Interpreter can read the provided model.
   tflite::ops::builtin::BuiltinOpResolver resolver;
-  InterpreterBuilder builder(*model, resolver);
-  std::unique_ptr<Interpreter> interpreter;
+  tflite::InterpreterBuilder builder(*model, resolver);
+  std::unique_ptr<tflite::Interpreter> interpreter;
```

Comment:
It is curious how the original file is inconsistent on using the `tflite::` namespace. The top of the file has `using namespace tflite;` so my inclination would be either remove that line and use `tflite::` explicitly, or remove the `tflite::` throughout and leave the `using` statement. So, if you prefer to keep the `tflite::` throughout, please remove the `using namespace tflite;` statement on line 31.

Reply:
I agree! I've removed the using statement now.

Reply:
I agree! I've removed the `using` statement now.